app/javascript/packs/one-time-code-input.js

```diff
@@ -1,6 +1,18 @@
 import OneTimeCodeInput from '@18f/identity-one-time-code-input';
 
-const input = document.querySelector('.one-time-code-input');
-if (input) {
-  new OneTimeCodeInput(/** @type {HTMLInputElement} */ (input)).bind();
+const fakeField = /** @type {HTMLInputElement?} */ (document.querySelector('.one-time-code-input'));
+
+if (fakeField) {
+  const hiddenInput = /** @type {HTMLInputElement} */ (document.createElement('input'));
+  hiddenInput.name = fakeField.name;
+  hiddenInput.value = fakeField.value;
+  hiddenInput.type = 'hidden';
+  /** @type {HTMLElement} */ (fakeField.parentNode).insertBefore(hiddenInput, fakeField);
+  fakeField.removeAttribute('name');
+  if (hiddenInput && fakeField) {
+    fakeField.addEventListener('input', () => {
+      hiddenInput.value = fakeField.value;
+    });
+  }
```

Comment:
We shouldn't need the condition, since we already checked `fakeField` on line 5, and created `hiddenInput` on line 6.

```suggestion
  fakeField.addEventListener('input', () => {
    hiddenInput.value = fakeField.value;
  });
```